kitchen/wazuh-puppet/dockerfile

```diff
@@ -1,14 +1,19 @@
 FROM vpgrp/kitchen:latest
 
 ARG source_branch=master
+ARG target_branch=master
 
 # Update aptitude with new repo
 RUN apt-get update
 
 # Clone the corresponding branch of wazuh-puppet
 RUN git clone https://github.com/wazuh/wazuh-puppet.git
 
-RUN cd wazuh-puppet && git checkout $source_branch && git branch && git pull
+RUN cd wazuh-puppet && \
+    git pull --all && \
```

Comment:
Why `git pull` here?

Reply:
Hi Manu!

The purpose here is to update the tracking info for all the branches.

Kr,

Rshad

Reply:
I think we should better use `git fetch --all` here.